src/main/java/fact/extraction/AboveThreshold.java

```diff
@@ -37,25 +42,29 @@ public Data process(Data input) {
 
         final int npix = (Integer) input.get("NPIX");
         final int roi = (Integer) input.get("NROI");
-        final short[] timeSerieses = (short[]) input.get(dataKey);
+        final short[] timeSeries = (short[]) input.get(dataKey);
```

Comment:
I propose to replace all timeseries(es) with timelines  or sth. else which has a sane plural representation. We should not limit ourselves by using variable names which have no flexibility or restricted features.